akka-actor-tests/src/test/scala/akka/io/TcpConnectionSpec.scala

```diff
@@ -911,6 +911,7 @@ class TcpConnectionSpec extends AkkaSpec("""
       new ChannelRegistration {
         def enableInterest(op: Int): Unit = interestCallReceiver.ref ! op
         def disableInterest(op: Int): Unit = interestCallReceiver.ref ! -op
+        def wakeUp(): Unit = ()
```

Comment:
This addition is reported as a potential binary incompatibility - presumably because someone could have extended this class? That seems fairly safe to ignore I suppose?